src/lib/pageNames.ts

```
@@ -0,0 +1,86 @@
+export type PageNameInfo = {
+  pageName: string
+  bookmarkPath?: string[]
+  pageLabel?: string
+}
+
+const ILLEGAL = /[<>:"/\\|?*\u0000-\u001F]/g
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)